book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental) unless @rentals.include?(rental)
+    rental.book = self
+  end
```

Comment:
- [ ] in order to set up a many to many relationship the Rental should be initialized here passing the date, the person and the book, so it will look something like below 
```
def add_rental(person, date) 
  Rental.new(date, self, person)
end

```

Reply:
Hello, the way my code is structured the `add_rental` method is supposed to ad a rental to `@rentals` that the person has. the rental will already have been created by whoever is calling the method `add_rental`